### PR TITLE
Add PredictedLabel to the Anomaly Detection task

### LINKED ISSUE #13481: ML.NET Anomaly Detection Task output should include PredictedLabel
The [Anomaly detection inputs and outputs](https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection-inputs-and-outputs) section should have [`PredictedLabel` in addition to `Score`](https://github.com/dotnet/machinelearning/blob/master/src/Microsoft.ML.PCA/PcaTrainer.cs#L381-L385).  Note: There is an [issue](https://github.com/dotnet/machinelearning/issues/3990) with the current release of ML.NET where `PredictedLabel` always evaluates to `true`.

Additionally, the description for `Score` reads:
> The non-negative, unbounded score that was calculated by the anomaly detection model.

This does not indicate how the value returned by the model for `Score` should be interpreted.  Should higher scores be interpreted as anomalies, or scores closer to zero?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bd15175c-5b7d-4310-ecf5-32ffc7bc588b
* Version Independent ID: 38adc676-934e-0ff9-c868-527f075c96df
* Content: [Machine learning tasks - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#anomaly-detection)
* Content Source: [docs/machine-learning/resources/tasks.md](https://github.com/dotnet/docs/blob/master/docs/machine-learning/resources/tasks.md)
* Product: **dotnet-ml**
* GitHub Login: @natke
* Microsoft Alias: **johalex**

### PR DESCRIPTION
**Pull request by natke**
Fixes #13481

Preview link: https://review.docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks?branch=pr-en-us-16388#anomaly-detection-inputs-and-outputs

### CODE REVIEW

File: docs/machine-learning/resources/tasks.md [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
-ms.date: 04/23/2019
+ms.date: 12/23/2019
 author: natke
 ---
 # Machine learning tasks in ML.NET
 
-When building a machine learning model, you first need to define what you are hoping to achieve with your data. This allows you to choose the right machine learning task for your situation. The following list describes the different machine learning tasks that you can choose from and some common use cases. For more information about choosing the task that is appropriate for your scenario, see [Algorithms](../how-to-choose-an-ml-net-algorithm.md).
+A machine learning task is determined by the type of problem you are trying to solve and the data you have.
+
+The following list describes the different machine learning tasks that you can choose from and some common use cases. For more information about choosing the task that is appropriate for your scenario, see [Algorithms](../how-to-choose-an-ml-net-algorithm.md).
```
Comment (luisquintanilla) [inline docs/machine-learning/resources/tasks.md]: In this paragraph, the wording in the last sentence may be confusing. If I am in the Tasks doc, why do I need to go to the Algorithms doc to help me choose a task?

**Review by luisquintanilla — COMMENTED** [review]
Looks good. Slight clarification might be needed in intro paragraphs.

File: docs/machine-learning/resources/tasks.md (hunk 1 above)
Comment (natke) [reply docs/machine-learning/resources/tasks.md]: Yes I agree. I've clarified the text. There is also an item on the backlog to completely refactor this doc to better explain machine learning tasks.

**Review by luisquintanilla — APPROVED** [review]
Looks good to me!

**Pull request merged**